content/php/concepts/string-functions/terms/html-entity-decode/html-entity-decode.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: 'PHP html_entity_decode() Function'
+Description: 'The html_entity_decode() function converts HTML entities to characters.'
+Subjects: 
+- 'PHP'
```

Comment:
PHP is not present here in [subjects](https://github.com/Codecademy/docs/blob/main/documentation/subjects.md). Choose something that's present in the subject, please.